scripts/update_specs_from_pricing.py

```diff
@@ -49,6 +49,8 @@
     'US West (Oregon)': 'us-west-2',
     'US West (Los Angeles)': 'us-west-2',
     'US West (Verizon) - San Francisco Bay Area': 'us-west-2',
+    'US East (Verizon) - Miami': 'us-east-1',
+    'US East (Verizon) - Dallas': 'us-east-1',
```

Comment:
Wondering if it'd be worth supporting patterns if this list keeps growing
```suggestion
    'US West (Verizon) - .*': 'us-west-2',
    'US East (Verizon) - .*': 'us-east-1',
```

Reply:
was starting to debate that too.  I wonder if we can just drop the verizon data?  I don't think it adds anything either.

Reply:
https://github.com/aws-cloudformation/cfn-python-lint/pull/1793